application/features/mfa.feature

```diff
@@ -120,3 +119,29 @@ Feature: MFA
       And 0 codes should be stored
       And the MFA record is not stored
 
+  Scenario: Exception to delete the missing credential of a webauthn MFA option
+    Given the user has a verified webauthn MFA with a key_handle_hash of "u2f"
+    When I request to delete the webauthn entry of the MFA with a webauthn_id of 999
+    Then the response status code should be 404
+
+  Scenario: Delete the legacy u2f credential of a webauthn MFA option
```

Comment:
The scenario description doesn't seem to match. Isn't it deleting a webauthn credential?